packages/commonwealth/client/scripts/models/IChainAdapter.ts

```diff
@@ -150,6 +160,26 @@ abstract class IChainAdapter<C extends Coin, A extends Account> {
     }
   }
 
+  public getTopicThreshold(topicName: string): BN {
+    if (this.gatedTopics?.length > 0 && topicName) {
+      const topicGate = this.gatedTopics.find((i) => i.topic === topicName);
+
+      if (!topicGate) return new BN('0', 10);
+
+      return new BN(topicGate.data.threshold);
+    }
+    return new BN('0', 10);
+  }
+
+  public isGatedTopic(topicName: string): boolean {
```

Comment:
id instead of name

Reply:
updated(note I only used topicName b/c that's the old function signature for isGatedTopic which made it plug and play. refactored no though